specs.json

```diff
@@ -447,14 +447,24 @@
       "eventHook": "object|string",
       "toolHook": "object|string",
       "events": "array",
-      "llmOptions": "object"
+      "llmOptions": "object",
+      "mcpServer": "#mcpServer"
```

Comment:
shouldnt this be an array of mcp servers?  People should be able to have more than a single mcp server

Reply:
fixed